templates/form/textarea.html.twig

```diff
@@ -13,6 +13,19 @@
  * @see template_preprocess_textarea()
  */
 #}
-<div{{ wrapper_attributes }}>
+
+{% wrapper_attributes.removeClass('form-textarea-wrapper') %}
+
+{% set wrapper_attributes_rendered %}
+  {{ wrapper_attributes }}
+{% endset %}
+
+{% if wrapper_attributes_rendered %}
+<div{{ wrapper_attributes_rendered }}>
```

Comment:
We _could_ add it back here in case there are actually other attributes and the wrapper prints out, it'd be good if it had a class.